db/migrate/20190118204247_move_miq_set_memberships.rb

```diff
@@ -0,0 +1,38 @@
+require 'ancestry'
+
+class MoveMiqSetMemberships < ActiveRecord::Migration[5.0]
+  class Relationship < ActiveRecord::Base
+    has_ancestry
+  end
+
+  class MiqSetMembership < ActiveRecord::Base
+  end
+
+  def up
+    say_with_time("Converting set relationships to miq_set_memberships") do
+      # ancestry holds the id for the relationship record that references the set
+      # the parent record contains the set object
+      # because these are trivial trees, we can look up the parent and get the resource_id
```

Comment:
sorry - I probably told you to write that last sentence and here I am changing again.

```suggestion
      # because these are trivial trees, we can get the set_id from the parent's resource_id
```